kafka-bom/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+plugins {
+  `java-platform`
+  id("org.hypertrace.publish-plugin")
```

Comment:
We haven't tried publishing a platform before - let me go check and make sure our publish plugin supports it.

Reply:
It doesn't, updated here - https://github.com/hypertrace/hypertrace-gradle-publish-plugin/pull/25